### PR TITLE
updated site with your apps name change

### PR DESCRIPTION
**Pull request by jpvajda**
## Description
- changed documentation to swap out Other apps for Your apps
- updated screen shots 
- fixed json formatting that was causing a page to scroll due to improper use of line breaks

## Reviewer Notes
This isn't to be merged until the corresponding PR in the Nr1-Catalog Repo is merged in.

## Related Issue(s) / Ticket(s)

DEVEX-1296

### CODE REVIEW

**Comment by jpvajda** [comment]
This change has occurred on production in the NR Catalog, so this is ok to review / merge now

File: src/data/related-pages.json [hunk 1]
```diff
@@ -162,11 +162,7 @@
     {
       "image": "",
       "url": "https://developer.newrelic.com/components/platform-state-context/",
-      "sections": [
-        "PlatformStateContext",
-        "Usage",
-        "Props"
-      ],
+      "sections": ["PlatformStateContext", "Usage", "Props"],
```
[the remark this answers is not in the document]
Comment (jpvajda) [reply src/data/related-pages.json:165]: perhaps.. hmmm

File: src/markdown-pages/build-apps/howto-use-nrone-table-components.mdx [hunk 2]
```diff
@@ -38,18 +37,22 @@ In this guide, you are going to build a sample table using various New Relic One
 ## Before you begin
 
 If you haven't already installed the New Relic One CLI, step through the [quick start in New Relic One](https://one.newrelic.com/launcher/developer-center.launcher). This process also gets you an API key.
-<br/>
-In addition, to complete the steps in this guide, you need a GitHub account, and to have Node.js installed on your machine. See [Setting up your development environment](/build-apps/set-up-dev-env) for more info. 
+
+<br />
+In addition, to complete the steps in this guide, you need a GitHub account, and
+to have Node.js installed on your machine. See [Setting up your development environment](/build-apps/set-up-dev-env)
```
[the remark this answers is not in the document]
Comment (jpvajda) [reply src/markdown-pages/build-apps/howto-use-nrone-table-components.mdx:43]: i'll fix.

[the remark this answers is not in the document]
Comment (jpvajda) [reply src/markdown-pages/build-apps/howto-use-nrone-table-components.mdx:43]: I figured it out, it's in between to <br> tags which seemed to mess with the markdown functionality.

File: src/data/related-pages.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpvajda) [reply src/data/related-pages.json:165]: Yeah my linter just flattens that one array even when i format it correctly manually.

File: src/markdown-pages/build-apps/publish-deploy.mdx [hunk 3]
```diff
@@ -28,6 +28,8 @@ try it out locally to make sure it's working properly.
 
 </Intro>
 
+<br />
```
Comment (jpvajda) [inline src/markdown-pages/build-apps/publish-deploy.mdx:31]: I added a <br> tag here to add more space between the image

File: src/data/related-pages.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (zstix) [reply src/data/related-pages.json:165]: There are too many changes in this file to reliably review and it looks like most of them are your editor adjusting the file. Would you mind backing out what your linter changed or help us in resolving the merge conflicts? @jpvajda

[the remark this answers is not in the document]
Comment (jpvajda) [reply src/data/related-pages.json:165]: @zstix will do. it seems the linter flattened all the arrays to one line. Not sure why it did that..

**Comment by jerelmiller** [comment]
@jpvajda I should also note that you shouldn't ever change that `src/data/related-pages.json`. That page is generated every time the cron job kicks off to pull new results from Swiftype. Its automatically updated by a script.

If you need changes to that file, then you'll need to manually kick off the script to re-index that page (@zstix should be able to show you how to do it in GH actions). I'd recommend resetting that file to what's on the `main` branch. (`git fetch && git checkout origin/main -- src/data/related-pages.json`)

**Comment by jerelmiller** [comment]
> you'll need to manually kick off the script to re-index that page

Edit: you'll need to manually kick off the script **after merging to `main`** to re-index that page

**Comment by jpvajda** [comment]
Closing this PR and starting from the beginning...

**Pull request closed without merging**